.github/workflows/integration-tests.yml

```diff
@@ -475,11 +445,22 @@ jobs:
           this-job-name: ETH Smoke Tests ${{ matrix.product.name }}${{ matrix.product.tag_suffix }}
           test-results-file: '{"testType":"go","filePath":"/tmp/gotest.log"}'
         continue-on-error: true
-      - name: Keep action running to view traces
+      - name: Wait for OTEL-Collector to flush buffered traces
         if: steps.check-label.outputs.trace == 'true' && matrix.product.name == 'ocr2' && matrix.product.tag_suffix == '-plugins'
         run: |
-          echo "Sleeping for $SLEEP_TIME seconds..."
-          sleep $SLEEP_TIME
+            sleep 60
+      - name: Permissions on traces
+        if: steps.check-label.outputs.trace == 'true' && matrix.product.name == 'ocr2' && matrix.product.tag_suffix == '-plugins'
+        run: |
+            ls -l ./integration-tests/smoke/traces
+            # chmod -R 777 ./integration-tests/smoke/traces
```

Comment:
commented out?

Reply:
obsoleted by changing the docker user for the otel-collector to match the user of the runner